.github/workflows/deploy.yml

```diff
@@ -25,13 +25,14 @@ jobs:
         with:
           role-to-assume: ${{ secrets.AWS_ROLE_ARN }}
           docker-repo: ${{ secrets.DOCKER_REPO }}
-      - uses: mbta/actions/deploy-scheduled-ecs@v2
+      - uses: mbta/actions/deploy-ecs@v2
         with:
           role-to-assume: ${{ secrets.AWS_ROLE_ARN }}
           ecs-cluster: linux-${{ inputs.environment == 'prod' && 'prod' || 'staging' }}
           ecs-service: uncooked-gps-${{ inputs.environment || 'dev' }}
           ecs-task-definition: uncooked-gps-${{ inputs.environment || 'dev' }}
           docker-tag: ${{ steps.build-push.outputs.docker-tag }}
+          launch-type: EXTERNAL
```

Comment:
I noticed that [Trike specifies launch-type in the action explicitly](https://github.com/mbta/trike/blob/main/.github/workflows/dev.yml#L36C11-L36C32), so I'm adding that here...